src/data_model/utils.jl

```diff
@@ -535,7 +535,7 @@ function _pad_properties_delta!(object::Dict{String,<:Any}, properties::Vector{S
             merge!(tmp, Dict((k[2], k[1])=>sign*v for (k,v) in tmp))
             get_val(x,y) = haskey(tmp, (x,y)) ? tmp[(x,y)] : 0.0
 
-            object[property] = [get_val(phases[1], phases[2]), get_val(phases[2], phases[3]), get_val(phases[3], phases[1])]
+            object[property] = val_length==1 ? [get_val(connections[1], connections[2]), 0.0,0.0] : [get_val(phases[1], phases[2]), get_val(phases[2], phases[3]), get_val(phases[3], phases[1])]
```

Comment:
I'm not clear on why there are four numbers for the `val_length == 1` case, instead of 3 like in the original case

Reply:
`get_val(x,y)` gives one number, which is basically `tmp[(connections[1], connections[2])]` (or `val[1]` as per line 528). The next two numbers are always zeros for single-phase delta loads. So, I think `object[property]` should still have only three numbers.